keras_cv/layers/preprocessing/rgb_shift.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class RGBShift(layers.Layer):
+    """Randomly shift values for each channel of the input image(s).
+
+    The input images should have values in the `[0-255]` range.
```

Comment:
lets add a value_range parameter like the other color based layers now have

Reply:
I think it doesn't require that much here. It's assumed that all input would be in the range of `[0, 255]`.

Reply:
We can't assume all inputs in [0, 255].  We do need a value range to be consistent with the other ones.  Otherwise all other KPLs that rely on color values will support value_range=(0, 1) and this one will silently fail for users in that case.